plugins/destination/file/client/client_test.go

```diff
@@ -62,10 +75,12 @@ func TestPluginParquet(t *testing.T) {
 			SkipOverwrite:             true,
 			SkipDeleteStale:           true,
 			SkipSecondAppend:          true,
-			SkipMigrateAppend:         true,
```

Comment:
Parquet format doesn't support any changes or appends once the file footer is written.

Reply:
OK so the tests API is a bit confusing but this will be skipped actually since the migrate strategy is forced:
https://github.com/cloudquery/plugin-sdk/blob/fac9ed3681d4f10800274a6bd35d2fa9cdcc732d/plugins/destination/plugin_testing_migrate.go#L91

Once we add support for forced mode (which will probably just delete the existing file), we'll need to remove `SkipMigrateAppendForce` and then it will verify the forced append work 🙃